### PR TITLE
Add `package.xml` to be listed in the FreeCAD Addon Manager

### LINKED ISSUE #514: Add to the FreeCAD Addon Manager
* **I'm submitting a ...**
  - [x] feature request

* **What is the motivation / use case for changing the behavior?**
	* Streamlining installation of 3rd party dependencies 
	* Exposure of the addon to the FreeCAD community at large

### Prerequisites
- [ ] A `package.xml` file is necessary https://wiki.freecadweb.org/Package_Metadata
- [ ] 3rd party dependencies need to be reviewed to see how they can be installed

The FreeCAD Addon Manager has its own repository at https://github.com/FreeCAD/FreeCAD-addons/

### PR DESCRIPTION
**Pull request by luzpaz**
Closes #514

* **Please check if the PR fulfills these requirements**
- [ ] The commit messages follows our guidelines (explicit, in english)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix:
- [x] New features: xml file that will allow this workbench to be listed in to the FC Addon Manager
- [ ] Documentation
- [ ] Optimization

* **Other information**:

This PR is still in process and requires input from the devs.  
- [ ] Make sure 3rd party deps are accurate
- [ ] Update all remaining placeholders in `package.xml`
- [ ] Missing InitGui.py 
- [ ] Missing SVG icon for the workbench

### CODE REVIEW

**Comment by pgibertini** [comment]
Hello @luzpaz you should update the changelog in order to pass the checks in CI

**Comment by luzpaz** [comment]
> Hello @luzpaz you should update the changelog in order to pass the checks in CI

Done in 2c67e90

File: package.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<package format="1" xmlns="https://wiki.freecad.org/Package_Metadata">
+  <name>Volmdlr Workbench</name> <!-- What the Addon Manager displays to users -->
+  <description>A computations-oriented python VOLume MoDeLeR with STEP support for import and export.</description>
+  <version>0.9.0</version> <!-- Semantic versioning (1.2.3-beta) or CalVer-based, (2022.01.07), don't omit or non-git installations won't see your updates -->
+  <date>2023-03-06</date> <!-- Date of the last update to the version number -->
+  <maintainer email="your_address@null.com">Your Name</maintainer>
```
Comment (luzpaz) [inline package.xml:7]: Please advise

File: package.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  <license file="LICENSE">GPL-3</license> <!-- Make sure you actually have this file in your Addon repo if the license requires it -->
+  <url type="repository" branch="master">https://github.com/Dessia-tech/volmdlr</url> <!-- Don't forget to update the branch name here -->
+  <url type="readme">https://github.com/Dessia-tech/volmdlr/blob/master/README.md</url> <!-- Link to the HTML-rendered README page -->
+  <icon>path/to/icon.svg</icon> <!-- If you include your icon here, you don't have to submit it to the main FreeCAD repo -->
```
Comment (luzpaz) [inline package.xml:11]: Is it possible for this workbench to have an icon that is in SVG format, it makes scaling seamless on HiDPi screens and therefore preferable to display in the Addon Manager (and in the workbench in general)

File: package.xml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      <depend>numpy</depend>
+      <depend>scipy</depend>
+      <depend>jsonschema</depend>
+      <depend>networkx</depend>
+      <depend version_lte="0.10.9">plot_data</depend>
+      <depend>rtree</depend>
+      <depend>kaitaistruct</depend>
+      <depend>sympy</depend>
+      <depend>trimesh</depend>
+      <depend>triangle</depend>
```
Comment (luzpaz) [inline package.xml:32]: Are these all the correct 3rd party deps that this workbench requires ?

Comment (luzpaz) [reply package.xml:32]: bump

**Comment by luzpaz** [comment]
I'm not sure mine both request for comments was responded to. Please double check

File: package.xml (hunk 2 above)
Comment (masfaraud) [reply package.xml:11]: We don't have an icon for the moment, I keep this in mind

**Comment by luzpaz** [comment]
bump

**Comment by masfaraud** [comment]
done some changes in https://github.com/Dessia-tech/volmdlr/pull/750

**Pull request merged**